components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -100,20 +101,23 @@ export function WorkspaceEntry({ desc, model }: { desc: WorkspaceInfo, model: Wo
         setChangesModalVisible(true);
     }
     return <div>
-        <a className="rounded-xl whitespace-nowrap flex space-x-2 py-6 px-6 w-full justify-between hover:bg-gray-100 focus:bg-gitpod-kumquat-light cursor-pointer group" href={startUrl.toString()}>
+        <div className="rounded-xl whitespace-nowrap flex space-x-2 py-6 px-6 w-full justify-between hover:bg-gray-100 focus:bg-gitpod-kumquat-light group">
             <div className="pr-3 self-center">
-                <div className={stateClassName}>
-                    &nbsp;
-            </div>
+                <Tooltip content={state}>
+                    <div className={stateClassName}>
+                    </div>
+                </Tooltip>
             </div>
             <div className="flex flex-col w-3/12">
-                <div className="font-medium text-gray-800 truncate">{ws.id}</div>
-                <a href={project ? 'https://' + project : undefined}><div className="text-sm overflow-ellipsis truncate text-gray-400">{project || 'Unknown'}</div></a>
+                <a href={startUrl.toString()}><div className="font-medium text-gray-800 truncate hover:text-blue-500">{ws.id}</div></a>
```

Comment:
**thought**: The idea was to have a rounded background on hover as in the pending changes column. Blue is not a color we've used for links elsewhere but let's get this is and fine tune within this week.

Reply:
We decided to go with blue-600.